packages/@uppy/transloadit/src/Client.ts

```diff
@@ -159,7 +159,7 @@ export default class Client<M extends Meta, B extends Body> {
     }
     const size = encodeURIComponent(file.size!)
     const uploadUrl = encodeURIComponent(file.uploadURL)
-    const filename = encodeURIComponent(file.name)
+    const filename = encodeURIComponent(file.name ?? 'Unnamed')
```

Comment:
Hm it looks like we have to repeat "Unnamed" in a lot of places.
And in general, we have to think about the possibility that `.name` can be absent.

Have you considered injecting "Unnamed" as a `.name` in `GooglePhotos.tsx`?